examples/mnist-inference-web/README.md

```diff
@@ -27,14 +27,9 @@ makes it possible to build and run the model with the `wasm32-unknown-unknown` t
 special system library, such as [WASI](https://wasi.dev/). (See [Cargo.toml](./Cargo.toml) on how to
 include burn dependencies without `std`).
 
-For this demo, we use trained parameters (`model-4.json.gz`) and model (`model.rs`) from the
+For this demo, we use trained parameters (`model.bin`) and model (`model.rs`) from the
 [`burn` MNIST example](https://github.com/burn-rs/burn/tree/main/examples/mnist).
 
-During the build time `model-4.json.gz` is converted to
-[`bincode`](https://github.com/bincode-org/bincode) (for compactness) and included as part of the
-final wasm output. The MNIST model is initialized with trained weights from memory during the
-runtime.
-
 The inference API for JavaScript is exposed with the help of
 [`wasm-bindgen`](https://github.com/rustwasm/wasm-bindgen)'s library and tools.
 
```

Comment:
The state is not saved with f16 anymore, since it doesn't compile correctly without std, so I'll keep the link to issue 202 as potential improvement.

Reply:
> Probably you can also remove references to the wasm file since it is smaller, so that we don't have to keep updating it.

I'm not sure what you are refering to.

Reply:
Sorry I meant in the readme file, the document talks about the file size of the wasm output under comparison section (e.g. 1,509,747 bytes). 

It's okay. I'll update the readme file not to talk about the file sizes.